doc/base/classes.xml

```diff
@@ -388,7 +388,9 @@
 			<argument index="0" name="json" type="String">
 			</argument>
 			<description>
-				Parse json text to a Variant (use [method typeof] to check if it is what you expect).
+				Parse JSON text to a Variant (use [method typeof] to check if it is what you expect).
+
```

Comment:
If I'm to nitpick all the way through, this empty newline should also be removed ;) (it would be parsed as an empty paragraph).

BTW you may want to amend to commit log as it's no longer about Dictionary.